components/ViewHabits.tsx

```diff
@@ -35,35 +52,110 @@ const styles = {
     maxHeight: '16vh',
     maxWidth: '100vw',
   }),
+
   habitsContainer: css({
-    maxWidth: '500px',
+    maxWidth: '700px',
     margin: '0 auto',
   }),
-  addHabitButton: css({
-    borderRadius: '7px',
-    position: 'absolute',
-    padding: '0',
-    height: '48px',
-    width: '48px',
-    top: '65px',
-    right: '8px',
-    transform: 'translateY(-50%)',
+  weeklyContainer: css({
+    display: 'flex',
+    flexDirection: 'column',
+    border: '2.5px solid #fff',
+
+    tr: {
+      display: 'flex',
+      flexDirection: 'row',
+
+      justifySelf: 'space-between',
+      alignSelf: 'space-between',
+    },
+
+    td: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      flex: '1',
+      border: '0.5px solid #222',
+      padding: '0px 2px',
+      minHeight: '50px',
+      minWidth: '50px',
+      verticalAlign: 'middle',
+    },
+
+    th: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      flex: '1',
+      border: '0.5px solid #222',
+      backgroundColor: '#0066ff',
+      padding: '0px 2px',
+      minHeight: '50px',
+      minWidth: '50px',
+      verticalAlign: 'middle',
+    },
+
+    '& :first-of-type': {
+      flexBasis: '50%',
+    },
   }),
 };
 
+type TimeView = 'daily' | 'weekly';
+
 const ViewHabits = () => {
```

Comment:
This may be a great time to think about creating a different component for each type of view. I found it a little harder to determine which logic corresponds to which state!

Reply:
Split it into 2 separate components now.